pkg/dispatchcli/cmd/install.go

```diff
@@ -80,10 +81,10 @@ type imageConfig struct {
 type oauth2ProxyConfig struct {
 	ClientID     string `json:"clientID,omitempty" validate:"required"`
 	ClientSecret string `json:"clientSecret,omitempty" validate:"required"`
-	CookieSecret string `json:"cookieSecret,omitmepty" validate:"omitempty"`
+	CookieSecret string `json:"cookieSecret,omitempty" validate:"omitempty"`
 }
 type openFaasRepoConfig struct {
-	Host     string `json:"host,omitempty" validate:"required,hostname"`
+	Host     string `json:"host,omitempty" validate:"omitempty"`
```

Comment:
So, I'm not sure we want to default to docker hub.  In fact, sooner (hopefully very soon), we should deploy a registry with dispatch as a default (Harbor).  Optionally, you can point to an existing registry such as docker hub or artifcatory.

Reply:
We could then make it explicit. Like, specify the registry as 'index.docker.io' in order to use dockerhub?  I just want the 'host' configuration to be either an FQDN or nothing. Substituting a username in a host field is what I am trying to avoid. Thoughts?

Reply:
I'm ok with being explicit.  I think most of this really comes down to documentation.

Reply:
Let's then keep it required for now and explicitly specify index.docker.io to use dockerhub and when we have our own dispatch registry we can make this optional.